src/Organizations/OrganizationsList/OrganizationsList.js

```diff
@@ -51,13 +53,23 @@ const columnMapping = {
   orgStatus: <FormattedMessage id="ui-organizations.main.vendorStatus" />,
   isVendor: <FormattedMessage id="ui-organizations.main.isVendor" />,
 };
-const resultsFormatter = {
-  orgName: ({ name }) => name,
+const getResultsFormatter = ({ search }) => ({
+  orgName: (data) => (
```

Comment:
For consistency, I see that the parentheses are not used for 1 arg (lines 70-71)
```suggestion
  orgName: data => (
```